### PR TITLE
docs: update support matrix

### LINKED ISSUE #266: Update Support Matrix [BLOCKED until Catalyst VOD is in prod]
https://www.notion.so/livepeer/Studio-Release-Notes-Catalyst-VOD-version-1-0-7be615c2736a47b19852a1640d2062f4

### PR DESCRIPTION
**Pull request by ichung08**
resolves #266

### CODE REVIEW

**Review by 0xcadams — APPROVED** [review]
LGTM, may want to have @victorges take a look as well

File: docs/05-reference/support-matrix.mdx [hunk 1]
```diff
@@ -2,8 +2,41 @@
...
+:::note
+Only file sizes up to 1GB are supported.
+:::
+
+### Video
+
+| Container                    | Video codecs supported with container  |                                                                                                                                                                                   |
+| ---------------------------- | -------------------------------------- |
+| MPEG-4 (.mp4)                | AVC (H.264), HEVC (H.265)              |                                                                                                                                                                      |
+| QuickTime (.mov)             | AVC (H.264), HEVC (H.265)              |
```
Comment (victorges) [inline docs/05-reference/support-matrix.mdx:16]: We actually don't support mov yet so might wanna list mp4 only for now

File: docs/05-reference/support-matrix.mdx [hunk 2]
```diff
@@ -2,8 +2,41 @@
...
+| MPEG-4 (.mp4)                | AVC (H.264)                            |                                                                                                                                                                      |
+| HLS                          | AVC (H.264)                            |
+
+### Audio
+
+| Container     | Audio codecs supported with container |
+| ------------- | ------------------------------------- |
+| MPEG-4 (.mp4) | AAC                                   |
+
+## Ingest Support
```
Comment (victorges) [inline docs/05-reference/support-matrix.mdx:39]: WDYT of specifying "Livestream ingest"?

File: docs/05-reference/support-matrix.mdx [hunk 3]
```diff
@@ -2,8 +2,41 @@
 title: Support Matrix
 ---
 
+## Inputs
```
Comment (victorges) [inline docs/05-reference/support-matrix.mdx:5]: WDYT of better separating this doc overall between "on demand" (this) and "Livestream" (the ingest support)?

**Review by victorges — APPROVED** [review]
LGTM with some suggestions

File: docs/05-reference/support-matrix.mdx (hunk 3 above)
Comment (ichung08) [reply docs/05-reference/support-matrix.mdx]: addressed in newest commit

**Review by victorges — APPROVED** [review]
Nice!

**Pull request merged**